lib/kitchen/directions/bake_handbook/v1.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeHandbook
+  class V1
+    def bake(book:, title_element:)
+      book.pages('$.handbook').each do |page|
+        page.titles.each do |title|
+          title.replace_children(with:
+            <<~HTML
+              <span data-type="" itemprop="" class="os-text">#{title.text}</span>
+            HTML
+          )
+          title.name = title_element
+        end
+
+        # Create Outline Title
+        outline_html = <<~HTML
+          <div class="os-handbook-outline">
+            <h3 class="os-title">#{I18n.t(:handbook_outline_title)}</h3>
+          </div>
+        HTML
+        page.title.append(sibling:
+          <<~HTML
+            #{outline_html}
+          HTML
+        )
```

Comment:
this can just be `page.title.append(sibling: outline_html)` -- no need for string interpolation!